spec/lib/payment_calculator/turnip_steps/ecf/service_fees_steps.rb

```diff
@@ -13,11 +14,11 @@ class DummyClass
                       set_up_fee: @set_up_fee,
                       band_a: @band_a,
                       bands: [@band_a],
+                      uplift_amount: 100,
```

Comment:
If it's not been specified in the steps wouldn't zero be a better default? I can see this causing confusion later.

Reply:
That does seem to be the suitable default and if we were to expand the steps to use this dummy contract, I'd prefer a fast fail if for some reason it were checking uplift against declarations that didn't require uplift (e.g, Retained_1).